backend/app/core/browse.py

```diff
@@ -32,6 +32,7 @@ def to_search_resp_doc(row: dict) -> SearchResult:
         document_category=row["category"],
         document_geography=row["country_code"],
         # Stuff we don't use
+        document_sectors=[],  # FIXME: empty placeholder for tests to pass
```

Comment:
I wasn't sure how to deal with many sectors per document in RDS so have left this as a `FIXME`. Happy to be guided through it or for one of you to do it if it's quick :)

Reply:
Ah yeah, document sectors will need to be created separately, I'll have a look at this.

Reply:
We can do this afterwards though, no need to hold up the merge for this.